src/main/webapp/messages/messages_en.js

```diff
@@ -171,6 +171,22 @@ const en = {
     '(Other)':'(Other)',
     '(Please select a specialism)' : '(Please select a specialism)',
 
+    /* ===================== */
+    /* Request type */
+    /* ===================== */
+    'Soort aanvraag': 'Request type',
```

Comment:
Better to remove the translation from https://github.com/thehyve/dntp-portal/blob/master/src/main/webapp/app/services/RequestTypeService.js#L34 instead of translating back and forth. The translation already happens in the template when rendering.

Reply:
ok, I based it on SpecialismService, I thought it is maybe some workaround. In this case I will change the SpecialismService too.